src/main/java/cat/udl/eps/softarch/learnphysics/domain/Topic.java

```diff
@@ -23,10 +27,19 @@
     private String description;
 
     @NotBlank
-    //@OneToMany
-    private String questions;
+    @OneToMany
+    private List<Question> questions;
 
     @NotBlank
-    //@OneToMany
-    private String theory;
+    @OneToMany
+    private List<Theory> theory;
+
+    public Topic(int topicId, String name, String description, List<Question> questions, List<Theory> theory) {
```

Comment:
This constructor is doing nothing. You must assign every received parameter to the corresponding attribute.
Example:
`this.topicId = id;`